src/controllers/index.js

```diff
@@ -9,5 +10,7 @@ router.get('/', home.get);
 router.get('/signup', signup.get);
 router.post('/signup', signup.post);
 
```

Comment:
what is the need of this empty line??

please prettify your code so it's more readable